firmware/application/ui_record_view.cpp

```diff
@@ -107,13 +107,13 @@ uint32_t RecordView::set_sampling_rate(uint32_t new_sampling_rate) {
     auto oversample_rate = get_oversample_rate(new_sampling_rate);
     auto actual_sampling_rate = new_sampling_rate * toUType(oversample_rate);
 
-    /* We are changing "REC" icon background to yellow in BW rec Options >600kHz
-     * where we are NOT recording full IQ .C16 files (recorded files are decimated ones).
-     * Those decimated recorded files, has not the full IQ samples.
-     * are ok as recorded spectrum indication, but they should not be used by Replay app.
+    /* We are changing "REC" icon background to yellow in BW rec Options >1Mhz
```

Comment:
Personally I think 1MHz should still be yellow, as I was seeing 4-7% dropped samples at 1MHz.  Probably depends on the SD card.

Reply:
well, in my 2 Portapack ,in one with slow SD card , and it  happens what you mentioned, (but that one shows average avg writting  SD card , 3,1 MB/sec ,  then it is normal that can not handle 1MB REC (4MB/sec) ,  with that one , I just can REC correctly 750KB C16  or   1MB in C8 . ) 
![image](https://github.com/eried/portapack-mayhem/assets/86470699/0b025c16-0ea6-44a3-b7d8-b25517c83bc2)


With the other one , avg writing is  5.79 MB/sec,  and it works well ,(except latency issues) . 
But anyway , if you feel that we should put 1MB in yellow, we can do it . (I removed the yellow , because I could skip the M4 % cpu previous bottleneck , now all the botleneck is in the SD card interface. 
![image](https://github.com/eried/portapack-mayhem/assets/86470699/0460721c-96fa-4290-8ab8-7f7b7540530a)

Then in my opinion , I would leave 1MB not in yellow , unless it also did not work to you in C8 . (just my opinion) 
Cheers